manifests/base/application-controller-deployment/argocd-application-controller-service.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     targetPort: 8082
   - name: metrics
     protocol: TCP
-    port: 8084
-    targetPort: 8084
+    port: 8082
```

Comment:
is it intentional to change the metrics port?

Reply:
Yes, the port number for metrics is 8082 in statefulsets. Continuing to use the same.